source/core/ut_output_buffer_tmp.sql

```diff
@@ -0,0 +1,17 @@
+create table ut_output_buffer_tmp(
+  /*
+  * This table is not a global temporary table as it needs to allow cross-session data exchange
+  * It is used however as a temporary table with multiple writers.
+  * This is why it has very high initrans and has nologging
+  */
+  reporter_id    raw(32) not null,
+  message_id     number(38,0) not null,
+  text           varchar2(4000),
```

Comment:
minor quibble.. should you not have the nullable "long varchar" as the last column? that shoudl help with space usage..